src/ErrorHandler/ErrorHandler.php

```diff
@@ -19,7 +23,6 @@ final class ErrorHandler
     private string $memoryReserve = '';
     private bool $exposeDetails = true;
 
-    private LoggerInterface $logger;
     private ThrowableRendererInterface $defaultRenderer;
 
     public function __construct(LoggerInterface $logger, ThrowableRendererInterface $defaultRenderer)
```

Comment:
Shouldn't we remove logger from constructor because it's already in a setter?

Reply:
I don't think so. Then `$logger` be null in processing if `setLogger` won't be called.